src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/on_button_press_notification.cc

```diff
@@ -104,8 +104,10 @@ void OnButtonPressNotification::Run() {
     }
 
     // Send ButtonPress notification only in HMI_FULL or HMI_LIMITED mode
-    if ((mobile_api::HMILevel::HMI_FULL != app->hmi_level()) &&
-        (mobile_api::HMILevel::HMI_LIMITED != app->hmi_level())) {
+    const mobile_apis::HMILevel::eType app_hmi_level =
```

Comment:
Same comment as OnButtonEvent

Reply:
fixed in 9862c6687b716c086c446c99ff0df04b5402bd2c